src/components/Table/GridTable.tsx

```diff
@@ -241,10 +253,6 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
     string[],
   ] = useMemo(() => {
     function makeRowComponent(row: GridDataRow<R>, level: number): JSX.Element {
-      // We only pass sortState to header rows, b/c non-headers rows shouldn't have to re-render on sorting
-      // changes, and so by not passing the sortProps, it means the data rows' React.memo will still cache them.
-      const sortProps = row.kind === "header" ? { sortOn, sortState, setSortKey } : { sortOn };
```

Comment:
Nice!

Reply:
Maybe can remove `sortState` and `setSorKey` from this `useMemo`'s deps?

Reply:
🤦 - of course.